samples/SelfHostServer/SelfHostServer.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
+    <!-- TODO remove when https://github.com/dotnet/sdk/issues/396 is resolved -->
+    <RuntimeIdentifier Condition=" '$(TargetFramework)' != 'netcoreapp1.1' ">win7-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="App.config" />
```

Comment:
Ah, I see the `runtimeconfig.template.json` is only for netcoreapp? I thought it was used by both tfms. In that case, we'll need something like `<Content Include="<filename>" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" />` right?